.copr/Makefile

```diff
@@ -0,0 +1,17 @@
+srpm:
+	# Setup development environment
+	echo "Installing base development environment"
+	dnf install -y gdb make autoconf rpm-build gettext-devel automake libtool docker python3-paramiko python3-pyyaml
+	echo "Installing FreeIPA development dependencies"
+	dnf builddep -y --skip-broken -D "with_wheels 1" -D "with_lint 1" --spec freeipa.spec.in --best --allowerasing --setopt=install_weak_deps=False
```

Comment:
You don't need ``with_wheels`` or ``with_lint`` for RPM builds. The dependencies are only required for development and testing.

Reply:
Done